django-stubs/forms/models.pyi

```diff
@@ -36,7 +37,7 @@ from django.db.models import ForeignKey
 
 ALL_FIELDS: str
 
-_Fields = Union[List[Union[Callable, str]], Sequence[str], Literal["__all__"]]
+_Fields = Union[List[Union[Callable, str]], Collection[str], Literal["__all__"]]
```

Comment:
Btw, can we improve `Callable` type here? Right now it is just `Callable[..., Any]`. Maybe it can be more specific?

Reply:
As I wrote in the initial PR message, I don't see the code using `fields` or `exclude` as a callable anywhere, at least not in Django 3.2.4. Are you certain that's a feature that (still) exists?

Reply:
Ok, let's remove it then! 👍

Reply:
I pushed a commit containing its removal.